src/mappings/mappingHandlers.ts

```diff
@@ -65,6 +72,20 @@ export async function handleTransaction(tx: CosmosTransaction): Promise<void> {
   await txEntity.save();
 }
 
+export async function handleNativeTransfer(msg: CosmosMessage<NativeTransferMsg>): Promise<void> {
+  logger.info(`[handleMessage] (tx ${msg.tx.hash}): indexing message ${msg.idx + 1} / ${msg.tx.decodedTx.body.messages.length}`)
+  logger.debug(`[handleMessage] (msg.msg): ${JSON.stringify(msg.msg, null, 2)}`)
```

Comment:
Oh, quick - last minute thing:
```suggestion
  logger.info(`[handleNativeTransfer] (tx ${msg.tx.hash}): indexing message ${msg.idx + 1} / ${msg.tx.decodedTx.body.messages.length}`)
  logger.debug(`[handleNativeTransfer] (msg.msg): ${JSON.stringify(msg.msg, null, 2)}`)
```